src/spec/time_parser_spec.rb

```diff
@@ -0,0 +1,91 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require_relative '../time_parser'
+
+describe TimeParser do
+  subject { TimeParser.new }
```

Comment:
This is not necessary :) (Calling `subject` anywhere within this block [and nested blocks] will give you an instance of the described class)

Reply:
Cool, thanks will update!